Rover/Rover.h

```diff
@@ -136,7 +140,10 @@ class Rover : public AP_Vehicle {
 #if OSD_ENABLED || OSD_PARAM_ENABLED
     AP_OSD osd;
 #endif
-
+#if PRECISION_LANDING == ENABLED
+    AC_PrecLand precland;
+    AC_PrecLand_StateMachine precland_statemachine;
```

Comment:
Should this be here? sate machine shouldn't be in Rover

Reply:
Oops. Sorry, I missed it. Removed it now. Thanks!